vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -154,33 +148,63 @@ ent prj install --conflict-strategy=OVERRIDE
 ```
 
 ### Run a Project Locally
-1. Initialize Keycloak, which leverages docker-compose
+1. Initialize Keycloak, which leverages Docker Compose
 ``` sh
 ent prj ext-keycloak start
 ```
 2. Initialize backend microservices
 ``` sh
 ent prj be-test-run
 ```
+To fetch logs from bundle plugins, use the command
+```
+ent prj be-log
+```
+
 3. Initialize one or more frontend widgets, each from its own shell
 ``` sh
 ent prj fe-test-run
 ```
-See [this tutorial](../../tutorials/create/ms/run-local.md) for more details.
+Check out [Run Blueprint-generated Microservices and Micro Frontends in Dev Mode](../../tutorials/create/ms/run-local.md) for more details.
+
+### Bundle and Custom Resource Management
+Entando supports a series of `ent ecr` commands to manage bundles and custom resources. The following commands are particularly useful.
+
+Display the list of bundles associated with the current profile
+```
+ent ecr list
+```
+
+Generate a custom resource and deploy it to the current profile
+```
+ent ecr deploy
+``` 
+
+The helpers `get-bundle-id` and `get-plugin-id` have been added to `ent ecr` to calculate and display unique identifiers related to custom resources. This provides additional security controls around bundle-specific and microservice plugin resources.
+
+Determine the bundle identifier
+```
+ent ecr get-bundle-id
+```
+
+Determine the plugin identifier
+```
+ent ecr get-plugin-id
+```
 
 ### Get the Bundle ID and Plugin ID
-Entando will use a unique identifier for your bundle as a way to provide additional security controls around bundle-specific resources. You can determine this identifier with this command
+Entando uses a unique identifier for your bundle as a way to provide additional security controls around bundle-specific resources. You can determine this identifier with the command 
 ```sh
 ent prj get-bundle-id --auto
 ```
 
-If you have a microservice plugin in your project, a unique identifier will be calculated for it as well.
+A unique identifier will also be calculated for each microservice plugin in your project. You can determine this identifier with the command 
 ```sh
 ent prj get-plugin-id --auto --repo=<BUNDLE-REPO-URL>
 ```
 
 ## Bundle Commands
-Use the `ent bundler` command to prepare a bundle for publication or extract a bundle from an application
+Use the `ent bundler` command to prepare a bundle for publication or extract a bundle from an application.`
```

Comment:
Is there a single backtick  after the period?

Reply:
!!! i got backtick-happy